### ISSUE TITLE
Question on training the critics

### ISSUE THREAD

**Issue by jizongFox** [sKamiJ/DCCS#2]
Hello, 

Thanks for the great work and the comprehensive released code. 

I have a question regarding to the training of the critic in the code.
https://github.com/sKamiJ/DCCS/blob/b40d95d23f821713cbd8d4e307522145e8977799/train.py#L216
In the prior matching step, you disable all gradients from the encoder and just train the critic. There is no gradient flow backpropagated to the encoder and how do you realize the prior distribution matching?

Thanks.

**Comment by jizongFox** [comment]
I found the answer, thanks!
